### PR TITLE
fix: Created RuntimeService from BootstrapUtils

### PR DESCRIPTION
**Pull request by fboucquez**
Created RuntimeService from BootstrapUtils. One PR to start splitting the classes.

### CODE REVIEW

File: src/service/Utils.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        }
+    }
+
+    public static validateIsTrue(value: boolean, message: string): void {
+        if (!value) {
+            throw new Error(message);
+        }
+    }
+    public static logSameLineMessage(message: string): void {
+        process.stdout.write(RuntimeService.isWindows() ? '\x1b[0G' : '\r');
```
Comment (Jaguar0625) [inline src/service/Utils.ts:24]: dependency on RuntimeService seems unnatural. probably better to move isWindows into this class.

File: test/service/RuntimeService.test.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
+import { expect } from 'chai';
+import { it } from 'mocha';
+import { LoggerFactory, LogType, RuntimeService } from '../../src';
+const logger = LoggerFactory.getLogger(LogType.Silent);
+const service = new RuntimeService(logger);
+describe('RuntimeService', async () => {
+    it('exec when valid', async () => {
```
Comment (Jaguar0625) [inline test/service/RuntimeService.test.ts:7]: does this test pass on windows?

**Review by Jaguar0625 — CHANGES_REQUESTED** [review]

File: src/service/Utils.ts (hunk 1 above)
Comment (fboucquez) [reply src/service/Utils.ts]: Created OSUtils

File: test/service/RuntimeService.test.ts (hunk 2 above)
Comment (Jaguar0625) [reply test/service/RuntimeService.test.ts:7]: cc @gimre-xymcity

Comment (fboucquez) [reply test/service/RuntimeService.test.ts:7]: Jumped over to windows and now all tests are running on windows. Afair,  Travis's windows vm doesn't support docker. Keen to migrate to github actions if have the time (maybe actions works better, need to revisit)

**Pull request merged**